client/src/components/Page/index.tsx

```diff
@@ -1,14 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { withStyles, createStyles } from '@material-ui/core/styles';
 import { /*type*/ WithStyles, Theme } from '@material-ui/core/styles';
+import Header from 'components/Header';
 
 const styles = (theme: Theme) =>
   createStyles({
     page: {
       display: 'flex',
       flex: 1,
       flexDirection: 'column',
-      alignItems: 'center',
+      alignItems: 'stretch',
```

Comment:
This was kinda messing me up because setting `width: 100%` would not include padding unless I used `box-sizing: border-box` but then my vertical padding would also go inside my box.

I think it's more natural to have the children of page `stretch` because `div`s normal fill the width of the container and on mobile this is probably what we want most often. I'm open to changing it back if you don't like this.

Reply:
nit: you might even be able to remove this prop. i think [the default](https://www.w3schools.com/CSSref/css3_pr_align-items.asp) seems to be `stretch`
Up 2 u tho

Reply:
true!